app/Kwality.UVault.Auth0/Models/Management.Token.cs

```diff
@@ -57,11 +58,23 @@ public int ExpiresIn
         set;
     }
 
+    // ReSharper disable once MemberCanBeInternal
+    [JsonPropertyName("token_type")]
+    public string TokenType
+    {
+        get;
+
+        [UsedImplicitly]
+        set;
+    }
+
     // NOTE: A token is expired one the amount of seconds (see "Expired In") is passed.
     //       To ensure that we don't use an expired token, a safety mechanism is built in.
     //       The time at which the token is used isn't the same as the time at which the token is checked.
-    public bool IsExpired(IDateTimeProvider dateTimeProvider)
+    internal bool IsExpired(IDateTimeProvider dateTimeProvider)
```

Comment:
Good change. By making the class public, this property did become public as well.
Very nice to mark it as `internal` to avoid exposing some inner details.